digid_eherkenning/oidc/models/digid.py

```diff
@@ -36,48 +34,40 @@ class Meta:
 
     @property
     def oidcdb_username_claim(self) -> ClaimPath:
-        return [self.identifier_claim_name]
+        return self.bsn_claim
 
 
 class DigiDMachtigenConfig(OpenIDConnectBaseConfig):
-    # TODO: support periods in claim keys
-    vertegenwoordigde_claim_name = models.CharField(
-        verbose_name=_("vertegenwoordigde claim name"),
-        default="aanvrager.bsn",
-        max_length=50,
-        help_text=_(
-            "Name of the claim in which the BSN of the person being represented is stored"
-        ),
+    # TODO: these default claim names don't appear to be part of any standard...
+    representee_bsn_claim = ClaimField(
+        verbose_name=_("representee bsn claim"),
+        default=ClaimFieldDefault("urn:nl-eid-gdi:1.0:LegalSubjectID"),
+        help_text=_("Name of the claim holding the BSN of the represented user."),
     )
-    gemachtigde_claim_name = models.CharField(
-        verbose_name=_("gemachtigde claim name"),
-        default="gemachtigde.bsn",
-        max_length=50,
+    authorizee_bsn_claim = ClaimField(
+        verbose_name=_("representee bsn claim"),
```

Comment:
Authorizee does sound funny, but at least it's clear as to what it means :p 
```suggestion
        verbose_name=_("authorizee bsn claim"),
```